rainmaker_orchestrator/server.py

```
@@ -1,426 +1,627 @@
-            "configured_models": ["kimi", "ollama"]
-        }), 200
+        workspace_path = Path(settings.workspace_path) / path
 
-    except Exception as e:
-        logger.error(f"Health check failed: {e}", exc_info=True)
-        return jsonify({
-            "status": "unhealthy",
-            "reason": str(e)
-        }), 503
+        if not workspace_path.exists():
+            raise HTTPException(status_code=404, detail="Path not found")
```

Ruff (0.14.10)

Abstract `raise` to an inner function

(TRY301)